lua/pixelui/core/cl_color.lua

```diff
@@ -130,4 +130,22 @@ function PIXEL.GetRainbowColor()
     lastCol = HSVToColor((time * 50) % 360, 1, 1)
 
     return lastCol
+end
+
+local colorMeta = FindMetaTable("Color")
+
+function colorMeta:Copy()
+    return PIXEL.CopyColor(self)
```

Comment:
Seems pretty unnecessary, just do `colorMeta.Color = PIXEL.CopyColor`?

Same applies for Color:Offset().